docs/transparent-validation/rte/v1.1.0/specification.md

```diff
@@ -281,3 +281,47 @@ located on RTE board or use OrangePi Zero original socket.
 ### Reset button
 
 Reset button (`SW1`) located near relay is used for resetting the RTE itself.
+
+### FAQ
+
+* **How to set GPIO states to flash SPI:**
+
```

Comment:
It would be good to add at the beginning a description of why we should change this option (another voltage for the flash chip).